### PR TITLE
Initial cleanup

### PR DESCRIPTION
**Pull request by rylandg**
I removed the `src` directory (it's a trap for users) and hid the start file. I also added a basic README that explains the importance and rationale behind the project.

### CODE REVIEW

File: .start-local-express.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const express = require('express');
```
Comment (vogre) [inline .start-local-express.js]: I think this could be in core `reshuffle`: opened https://github.com/reshufflehq/reshuffle/pull/317.

Comment (bergundy) [reply .start-local-express.js]: Nice @vogre

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "express": "^4.17.1"
   },
   "scripts": {
-    "start": "node src/start.js",
+    "start": "node .start-local-express.js",
```
Comment (vogre) [inline package.json]: can be `"start": "reshuffle-local-server"`.

**Comment by rylandg** [comment]
@vogre @bergundy I've switched to the new versions, replaced the hidden file added a license and .gitignore. I will hold off on adding CircleCI and linting for a separate commit. Thanks for awesome quick addition Vlad.

**Pull request merged**